packages/editor-ui/src/components/ApiKeyCreateOrEditModal.vue

```diff
@@ -11,24 +11,68 @@ import { useDocumentTitle } from '@/composables/useDocumentTitle';
 import { useApiKeysStore } from '@/stores/apiKeys.store';
 import { useToast } from '@/composables/useToast';
 import type { BaseTextKey } from '@/plugins/i18n';
-import type { ApiKeyWithRawValue } from '@n8n/api-types';
+import { N8nText } from 'n8n-design-system';
+import { DateTime } from 'luxon';
+import type { ApiKey, ApiKeyWithRawValue, CreateOrUpdateApiKeyRequestDto } from '@n8n/api-types';
+
+const EXPIRATION_OPTIONS = {
+	'7_DAYS': 7,
+	'30_DAYS': 30,
+	'60_DAYS': 60,
```

Comment:
This is quite many predefined options. Maybe we could drop 60?

Reply:
I copied the same options as Github has.